<div class="py-4" >
    <div class="container">
        <div class="header">
            <img src="../../../../assets/LOGo.png" />
            <div class="header-sub" >
               <div>
                    <p>17115 Red Oak DR, Ste 123,</p>
                    <p>Houston TX 77090</p>
               </div>
               <div>
                    <p> P: 561.635.0392 | F: 832.4130059</p>
               </div>
               <div>
                    <p>CLIA: 45D2257672</p>
                    <p>Lab Director: Dr. Kerollos Shaker</p>
               </div>
            </div>
        </div>
  
        <div class="gradient-background" >

        </div>

      <table class="table table-striped table-custom-transparent mt-4 mb-0"  style="border: 1px solid black; border-collapse: collapse;">
        <thead>
          <tr>
            <th style="border: 1px solid black; font-weight:500;"  >
                <span>Patient Name : </span>
                <span>Umair</span>
            </th>
            <th style="border: 1px solid black; font-weight:500;" >
                <span>Date of Birth : </span>
                <span>2199/11/2</span>
            </th>
            <th style="border: 1px solid black; font-weight:500;" >
                <span>Accession ID : </span>
                <span>7892810921</span>
            </th>
          </tr>
        </thead>
      </table>
  
      <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
        <thead>
          <tr>
            <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Test Methodology</th>
          </tr>
        </thead>
        <tbody>
          <tr style="background: #fff;" >
            <td style="border: 1px solid black; padding: 13px 20px;">
                Cancer Risk Factor Screening 34 gene panel screening performed at Elixir Diagnostics utilizes Next-Generation
                Sequencing technology using Nextera Flex chemistry on the IIIumina MiniSeq and NextSeq platforms. Genomic DNA
                is extracted from Buccal swabs (Wet) are tagemented and enriched for regions of interest using probes specific for
                each region specified in the BED (Browser Extensible Data) file. Positive (www.coriell.org) and Negative controls are
                included with each machine run to ensure the accuracy of amplicon preparation alongside targeted sequencing on
                coding regions and intronic/exonic boundaries of interest and precise finding of the variant on the positive control and
                negative control respectively. Exclusions from analysis are listed below Sequences obtained are then aligned against
                a human reference genome and variants such as SNVs (Single Nucleotide Variants) and Indels (Insertions and Dele -
                tions) are noted.For a detailed list of regions covered and comprehensive statistics by Elixir Diagnostics Cancer
                screen. Computational analysis and variant calling is performed by ipseity (www.ipseity.bio). Briefly, reads from the
                sequence output were aligned to the human reference genome (GRCh37) using the Burrows-Wheeler Aligner (BWA).
                Variants to the reference were called using the Genomic Analysis Tool Kit (GATK). The variants were annotated and
                filtered using the Elixir Diagnostics analysis work flow implementing the ACMG guidelines for interpretation of
                sequence variants. This includes comparison against the gnomAD population catalog of variants in 123,136 exomes,
                the 1000 Genomes Project Consortium’s publication of 2,500 genomes, the NCBIClinVar database of clinical asser -
                tions on variant’s pathogenicity and multiple lines of computational evidence on conservation and functional impact.
                The following databases and insilico algorithms are used to annotate and evaluate the impact of the variant in the
                context of human disease: 1000 genomes, gnomAD, ClinVar, OMIM, dbSNP, NCIB Ref Seq Genes, ExAC Gene
                Constraints, VSSIFT, VS PolyPhen2, PhyloP, GERP+, GeneSplicer, Max EntScan, NNSplice, PWM Splice Predictor.
                Analysis was reported using the HGVS nomenclature (www.hgvs.org/mutnomen)as implemented by custom transcript
                annotation algorithm. Only variants with an end ACMG classification of "Pathogenic" or "Likely Pathogenic" are
                reported and VUS/Benign/Likely Benign varinats are not reported. Classification of pathogenicity is consistently
                updated by the National Institutes of Health and the information found within this report is consistent with the current
                knowledge base as the knowledge base changes, so may clinical interpretation of variants. A
            </td>
          </tr>
        </tbody>
      </table>

      <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
        <thead>
          <tr>
            <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Genes Evaluated</th>
          </tr>
        </thead>
        <tbody>
          <tr style="background: #fff;" >
            <td style="border: 1px solid black; padding: 13px 20px;">
                APC, ATM, BAP1, BARD1, BMPR1A, BRCA1, BRCA2, BRIP1, CDH1, CDK4, CDKN2A, CHEK2, COL1A1, EPCAM, FBN1,
                GREM1, KIT, MITF, MLH1, MSH2, MSH6, MUTYH, NBN, NF1, PALB2, PMS2, POLD1, POLE, PTEN, RAD51C, RAD51D,
                SMAD4, STK11, TP53.
            </td>
          </tr>
        </tbody>
      </table>

      <table class="table table-striped table-custom mb-0" style="border: 1px solid black; border-collapse: collapse; border-top: none;">
        <thead>
          <tr>
            <th style="border: 1px solid black; padding: 13px 20px; border-top: none; font-weight: 400; font-size: 15px;">Test Limitations</th>
          </tr>
        </thead>
        <tbody>
          <tr style="background: #fff;" >
            <td style="border: 1px solid black; padding: 13px 20px;">
                Rare diagnostics errors may occur if these mutations occur within the priming sequencing regions Presence of a
                pathogenic/likely pathogenic variant does not guarantee that an individual will develop a Cancer, nor is the absence
                of such variants a guarantee that an individual will not develop a 'Cancer'. The results of this screen are meant strictly
                to guide a physician in the management of their patient's health. Any Likely pathogenic or Pathogenic variants
                detected in the report should be clinically correlated (As per Physicians Advice) and confirmed via the
                orthogonal testing platforms (Sanger Sequencing or PCR based tests).
            </td>
          </tr>
        </tbody>
      </table>
      
    </div>
  </div>
  